include/functions.h

```diff
@@ -633,12 +633,12 @@ s32 BgCheck_SphVsFirstDynaPoly(CollisionContext* colCtx, u16 xpFlags, CollisionP
 void CollisionHeader_GetVirtual(void* colHeader, CollisionHeader** dest);
 void func_800418D0(CollisionContext* colCtx, PlayState* play);
 void BgCheck_ResetPolyCheckTbl(SSNodeList* nodeList, s32 numPolys);
-u32 SurfaceType_GetCamDataIndex(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
-u16 func_80041A4C(CollisionContext* colCtx, u32 camId, s32 bgId);
-u16 SurfaceType_GetCameraSType(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
-u16 SurfaceType_GetNumCameras(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
-Vec3s* func_80041C10(CollisionContext* colCtx, s32 camId, s32 bgId);
-Vec3s* SurfaceType_GetCamPosData(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
+u32 SurfaceType_GetBgCamDataIndex(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
+u16 BgCheck_GetBgCamDataSettingImpl(CollisionContext* colCtx, u32 camId, s32 bgId);
+u16 BgCheck_GetBgCamDataSetting(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
+u16 BgCheck_GetBgCamDataNumVec3s(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
+Vec3s* BgCheck_GetBgCamDataVec3sImpl(CollisionContext* colCtx, s32 camId, s32 bgId);
+Vec3s* BgCheck_GetBgCamDataVec3s(CollisionContext* colCtx, CollisionPoly* poly, s32 bgId);
```

Comment:
in general, I dont think we need to mention the fact that the data is of type vec3s in these names. It can be more generally described as the bgcam data right? `GetBgCamData` seems fine on its own

and I still dont like `numData`, tho we havent really agreed on what to call it yet. imo count is fine, and `GetBgCamDataCount` makes sense

Reply:
I think I'd be fine with `count` then.

The reason I put `Vec3s` in the name of these functions is becuase `GetBgCamData` makes it sound like it's getting the struct `BgCamData` instead of the Vec3s data itself, `bgCamData.data`, which the mixup happens because of the two repeated uses of `data` in `bgCamData.data`.

Any suggestions?

Reply:
I guess I view the vec3s data as the actual data, and the struct `BgCamData` as a wrapper that holds that data and information about that data (setting and count)

if my view of that is wrong, then we should potentially come up with different names to make that separation more clear

Reply:
Interesting, I didn't think of it exactly like that but it makes sense. Maybe we could change:
`BgCamData` -> `BgCamDataHeader`
`data` -> `bgCamData`

Looking over how it's used, we wouldn't need to change any temps or function names, and I think it would still all make sense. Thoughts?